<launch>
    <node name="node_data_for_obstacle_detection" pkg="draconis_demo_datafeed" type="node_data_for_obstacle_detection" output="screen">

        <param name="topic_cloud_in" value="/$(env DOG_NAME)/l515/depth/color/points" type="str" />
        <param name="topic_cloud_out" value="/$(env DOG_NAME)/l515_cloud" type="str" /> 

        <param name="cloud_out_frame_id" value="$(env DOG_NAME)/base_link" type="str" />
        <param name="sampling_rate" value="4" type="double"/>

    </node>
</launch>